src/account/AccountPickScreen.js

```diff
@@ -36,7 +36,20 @@ class AccountPickScreen extends PureComponent<Props> {
   };
 
   handleAccountRemove = (index: number) => {
-    this.props.dispatch(removeAccount(index));
+    Alert.alert(
+      'Remove Account?',
```

Comment:
Let's be sure to wire these new user-facing strings up for translation, by using `_` (see the jsdoc on our `GetText` type for how to get access to that in a class component, and examples like `UserStatusScreen` or `LanguagePicker`) and adding the new strings to messages_en.json.

Also, let's go for the same capitalization, punctuation, and button style as we use for the logout confirmation in src/account-info/ProfileScreen.js.